src/engine/source/api/src/kvdb/handlers.cpp

```diff
@@ -164,18 +163,27 @@ api::Handler managerDump(std::shared_ptr<kvdbManager::IKVDBManager> kvdbManager,
         }
 
         const auto& eRequest = std::get<RequestType>(res);
+        uint32_t page = eRequest.has_page() ? eRequest.page() : DEFAULT_HANDLER_PAGE;
```

Comment:
Why we require exactly unit32_t instead of unsigned int?

Reply:
uint_32 was used because the page field proto definition used the uint32 type, but we can use unsigned int without problems.